prov/efa/src/efa_mr.h

```diff
@@ -49,6 +49,7 @@ struct efa_mr_peer {
 		int             neuron;
 		int             synapseai;
 	} device;
+	bool use_gdrcopy;
```

Comment:
I have a hunch that we do not need this flag - simply set `reserved = 0` should be enough. WDYT?

Reply:
That will disregard user device, though it does not actually matter.

Reply:
Use `ofi_cudaIsDeviceId` to check whether it's a device id or handle in the new revision.